### ISSUE TITLE
Out of memory

### ISSUE THREAD

**Issue by pomber** [blake-mealey/ansi-sequence-parser#1]
Found a sequence that makes node go out of memory:

```js
import { createAnsiSequenceParser } from "ansi-sequence-parser";

const parser = createAnsiSequenceParser();

const code = `28s[4B[46D[0m[?2004h`;

console.log(parser.parse(code));
```

After some time running, it fails with:

```
<--- Last few GCs --->

[17198:0x58d97f0]    40063 ms: Mark-sweep 4055.9 (4138.5) -> 4043.8 (4142.3) MB, 1608.6 / 0.0 ms  (average mu = 0.193, current mu = 0.014) allocation failure scavenge might not succeed
[17198:0x58d97f0]    42500 ms: Mark-sweep 4060.8 (4143.5) -> 4048.8 (4147.5) MB, 2413.1 / 0.0 ms  (average mu = 0.095, current mu = 0.010) allocation failure scavenge might not succeed


<--- JS stacktrace --->

FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
 1: 0xb06730 node::Abort() [node]
...
```

**Comment by blake-mealey** [comment]
Yikes, thanks for the report! I'll dig into it when I get a chance.

**Comment by blake-mealey** [comment]
This is fixed in v1.1.1 which is now available on npm!
